ArcadiaParties-SPA/src/app/header/header.component.html

```diff
@@ -0,0 +1,4 @@
+<span>{{ title$ | async }}</span>
```

Comment:
When you use flex it doesn't matter whether to use divs or spans, but divs are more commonly used.

Reply:
I've changed spans to divs